sleap/gui/widgets/video.py

```diff
@@ -1732,6 +1733,211 @@ def updateEdge(self, node: QtNode):
+    scale the overall bounding box.
+
+    Args:
+        rect: The :class:`QRectF` object which defines the non-scalable bounding box.
+        parent: The :class:`QtInstance` to encompass.
+
+    """
+
+    def __init__(self, rect, parent=None):
+        super().__init__(rect, parent)
+        if parent is not None:
+            self.box_width = parent.markerRadius
+            color_manager = parent.player.color_manager
+            int_color = color_manager.get_item_color(parent.instance)
+            self.int_color = QColor(*int_color)
+        else:
+            self.box_width = 7
+            self.int_color = Qt.white
+
+        self.parent = parent
+        self.origin = rect.topLeft()
+        self.ref_width = rect.width()
+        self.ref_height = rect.height()
+
+        box_pen = QPen(Qt.black)
+        box_pen.setCosmetic(True)
+        box_brush = QBrush(self.int_color)
+
+        # Create the edge boxes
+        self.top_left_box = QtWidgets.QGraphicsRectItem(parent=self)
+        self.top_left_box.setPen(box_pen)
+        self.top_left_box.setBrush(box_brush)
+        self.top_left_box.setOpacity(0.8)
+        self.top_left_box.setCursor(QCursor(Qt.DragMoveCursor))
+
+        self.top_right_box = QtWidgets.QGraphicsRectItem(parent=self)
+        self.top_right_box.setPen(box_pen)
+        self.top_right_box.setBrush(box_brush)
+        self.top_right_box.setOpacity(0.8)
+        self.top_right_box.setCursor(QCursor(Qt.DragMoveCursor))
+
+        self.bottom_left_box = QtWidgets.QGraphicsRectItem(parent=self)
+        self.bottom_left_box.setPen(box_pen)
+        self.bottom_left_box.setBrush(box_brush)
+        self.bottom_left_box.setOpacity(0.8)
+        self.bottom_left_box.setCursor(QCursor(Qt.DragMoveCursor))
+
+        self.bottom_right_box = QtWidgets.QGraphicsRectItem(parent=self)
+        self.bottom_right_box.setPen(box_pen)
+        self.bottom_right_box.setBrush(box_brush)
+        self.bottom_right_box.setOpacity(0.8)
+        self.bottom_right_box.setCursor(QCursor(Qt.DragMoveCursor))
+
+    def setRect(self, rect: QRectF):
+        # Update edge boxes along with instance box
+        super().setRect(rect)
+        x1, y1, x2, y2 = rect.getCoords()
+        w = self.box_width
+        self.top_left_box.setRect(QRectF(QPointF(x1, y1), QPointF(x1 + w, y1 + w)))
+        self.top_right_box.setRect(QRectF(QPointF(x2 - w, y1), QPointF(x2, y1 + w)))
+        self.bottom_left_box.setRect(QRectF(QPointF(x1, y2 - w), QPointF(x1 + w, y2)))
+        self.bottom_right_box.setRect(QRectF(QPointF(x2 - w, y2 - w), QPointF(x2, y2)))
+
+    def mousePressEvent(self, event):
+        """Custom event handler for pressing on an adjustable corner box.
+
+        This function recognizes that the user has begun resizing the instance and
+        stores relevant information about the bounding box before the transformation.
+        """
+        if event.button() == Qt.RightButton:
+            self.resizing = None
```

Comment:
It looks like you're using `self.resizing` in multiple places. Since it's a instance property, you should set it to its default value in the `__init__()` constructor above, i.e., add a `self.resizing = None` in the constructor.